salt/modules/cron.py

```diff
@@ -336,9 +336,16 @@ def list_tab(user):
                     comment = comment_line
                 else:
                     comment += '\n' + comment_line
-            elif len(line.split()) > 5:
+            elif line.find('=') > 0 and (' ' not in line or line.index('=') < line.index(' ')):
+                # Appears to be a ENV setup line
+                comps = line.split('=')
```

Comment:
Splitting on `'='` and then joining with `' '` below would effectively remove all equals signs from the value side.

eg:

```
VAR="string  with  spaces  ==  big problem"
```

With this method you will want to use the second optional argument to `split` (maxsplit) - the following should work:

```
comps = line.split('=', 1)
```